tests/pytorch/test_pytorch_autolog.py

```diff
@@ -43,6 +44,19 @@ def pytorch_model_without_validation():
     return trainer, run
 
 
+@pytest.fixture
+def pytorch_model_with_every_step_logged():
+    mlflow.pytorch.autolog(log_every_n_step=1)
```

Comment:
@adamreeve Can we add a test case for `log_every_n_step != 1` (say 10)?

Reply:
Yep good idea, I've added that now